### PR TITLE
fix failed test

### PR DESCRIPTION
**Pull request by vankichi**
Signed-off-by: vankichi <kyukawa315@gmail.com>

### Description:

I fixed below tests which are failed.
  - `internal/log/glg/glg_test`
  - `internal/servers/server/option_tset`

### Environment:

- Go Version: 1.14.3
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.11.5

### Types of changes:

- [x] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [x] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [x] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by kevindiu** [comment]
seems the CI still failed.

**Comment by vankichi** [comment]
It is because of panic pattern test 😓 
I think we have to make decision about it.

**Comment by kevindiu** [comment]
this error occurs...
```
# github.com/vdaas/vald/internal/io [github.com/vdaas/vald/internal/io.test]
io/io_test.go:429:28: not enough arguments in call to test.checkFunc
	have (want, io.Writer, error)
	want (want, io.Writer, string, error)
```

**Comment by kevindiu** [comment]
Also after I ran the go test command in `internal` folder, the `go.sum` file changed. Should we fix it also?
```
git diff ../go.sum
diff --git a/go.sum b/go.sum
index 47181348..6955a0c1 100644
--- a/go.sum
+++ b/go.sum
@@ -832,6 +832,7 @@ golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2/go.mod h1:tRJNPiyCQ0inRvYxb
 golang.org/x/time v0.0.0-20181108054448-85acf8d2951c/go.mod h1:tRJNPiyCQ0inRvYxbN9jk5I+vvW/OXSQhTDSoE431IQ=
 golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 h1:SvFZT6jyqRaOeXpc5h/JSfZenJ2O330aBsf7JfSUXmQ=
 golang.org/x/time v0.0.0-20190308202827-9d24e82272b4/go.mod h1:tRJNPiyCQ0inRvYxbN9jk5I+vvW/OXSQhTDSoE431IQ=
+golang.org/x/time v0.0.0-20191024005414-555d28b269f0 h1:/5xXl8Y5W96D+TtHSlonuFqGHIWVuyCkGJLwGh9JJFs=
 golang.org/x/time v0.0.0-20191024005414-555d28b269f0/go.mod h1:tRJNPiyCQ0inRvYxbN9jk5I+vvW/OXSQhTDSoE431IQ=
 golang.org/x/tools v0.0.0-20180525024113-a5b4c53f6e8b/go.mod h1:n7NCudcB/nEzxVGmLbDWY5pfWTLqBcC2KZ6jyYvM4mQ=
 golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e/go.mod h1:n7NCudcB/nEzxVGmLbDWY5pfWTLqBcC2KZ6jyYvM4mQ=
```

**Comment by vankichi** [comment]
> io/io_test.go:429:28: not enough arguments in call to test.checkFunc

I could not confirm this problem... 😕 
and, CI is success.

**Comment by kevindiu** [comment]
> > io/io_test.go:429:28: not enough arguments in call to test.checkFunc
> 
> I could not confirm this problem... 😕
> and, CI is success.

Sorry that I forgot to paste the command here. Here is the command and the result.
```
% go test ./internal/... -count=1
ok  	github.com/vdaas/vald/internal/backoff	1.810s
ok  	github.com/vdaas/vald/internal/cache	0.804s
ok  	github.com/vdaas/vald/internal/cache/cacher	0.986s
ok  	github.com/vdaas/vald/internal/cache/gache	1.228s
?   	github.com/vdaas/vald/internal/client	[no test files]
ok  	github.com/vdaas/vald/internal/client/agent/grpc	1.170s
ok  	github.com/vdaas/vald/internal/client/agent/rest	0.693s
ok  	github.com/vdaas/vald/internal/client/compressor	0.679s
ok  	github.com/vdaas/vald/internal/client/discoverer	1.857s
ok  	github.com/vdaas/vald/internal/client/gateway/vald/grpc	1.206s
ok  	github.com/vdaas/vald/internal/client/gateway/vald/rest	0.715s
ok  	github.com/vdaas/vald/internal/compress	0.925s
ok  	github.com/vdaas/vald/internal/config	0.736s
ok  	github.com/vdaas/vald/internal/core/converter/tensorflow	1.758s
?   	github.com/vdaas/vald/internal/core/ngt	[no test files]
ok  	github.com/vdaas/vald/internal/db/kvs/redis	0.470s
ok  	github.com/vdaas/vald/internal/db/nosql/cassandra	0.825s
ok  	github.com/vdaas/vald/internal/db/rdb/mysql	0.732s
?   	github.com/vdaas/vald/internal/db/storage/blob	[no test files]
ok  	github.com/vdaas/vald/internal/db/storage/blob/s3	0.994s
ok  	github.com/vdaas/vald/internal/db/storage/blob/s3/reader	0.554s
ok  	github.com/vdaas/vald/internal/db/storage/blob/s3/session	0.903s
ok  	github.com/vdaas/vald/internal/db/storage/blob/s3/writer	1.085s
# github.com/vdaas/vald/internal/io [github.com/vdaas/vald/internal/io.test]
internal/io/io_test.go:429:28: not enough arguments in call to test.checkFunc
	have (want, io.Writer, error)
	want (want, io.Writer, string, error)
ok  	github.com/vdaas/vald/internal/errgroup	4.445s
ok  	github.com/vdaas/vald/internal/errors	0.401s
ok  	github.com/vdaas/vald/internal/file	0.471s
ok  	github.com/vdaas/vald/internal/file/watch	0.504s
ok  	github.com/vdaas/vald/internal/info	0.431s
FAIL	github.com/vdaas/vald/internal/io [build failed]
ok  	github.com/vdaas/vald/internal/k8s	2.265s
ok  	github.com/vdaas/vald/internal/k8s/metrics/node	1.343s
ok  	github.com/vdaas/vald/internal/k8s/metrics/pod	3.410s
ok  	github.com/vdaas/vald/internal/k8s/node	3.809s
ok  	github.com/vdaas/vald/internal/k8s/pod	1.419s
ok  	github.com/vdaas/vald/internal/log	0.542s
ok  	github.com/vdaas/vald/internal/log/format	0.579s
ok  	github.com/vdaas/vald/internal/log/glg	0.558s
ok  	github.com/vdaas/vald/internal/log/level	0.595s
ok  	github.com/vdaas/vald/internal/log/logger	0.544s
ok  	github.com/vdaas/vald/internal/log/mock	0.500s
ok  	github.com/vdaas/vald/internal/log/retry	0.865s
ok  	github.com/vdaas/vald/internal/net	1.189s
ok  	github.com/vdaas/vald/internal/net/grpc	0.523s
ok  	github.com/vdaas/vald/internal/net/grpc/metric	0.480s
ok  	github.com/vdaas/vald/internal/net/grpc/pool	0.766s
ok  	github.com/vdaas/vald/internal/net/grpc/proto	0.866s
ok  	github.com/vdaas/vald/internal/net/grpc/status	0.777s
ok  	github.com/vdaas/vald/internal/net/http/client	1.070s
ok  	github.com/vdaas/vald/internal/net/http/dump	0.442s
ok  	github.com/vdaas/vald/internal/net/http/json	0.457s
ok  	github.com/vdaas/vald/internal/net/http/metrics	0.685s
ok  	github.com/vdaas/vald/internal/net/http/middleware	2.006s
ok  	github.com/vdaas/vald/internal/net/http/rest	0.510s
ok  	github.com/vdaas/vald/internal/net/http/routing	0.544s
ok  	github.com/vdaas/vald/internal/net/http/transport	0.540s
ok  	github.com/vdaas/vald/internal/net/tcp	0.578s
ok  	github.com/vdaas/vald/internal/observability	7.007s
ok  	github.com/vdaas/vald/internal/observability/client/google	0.664s
ok  	github.com/vdaas/vald/internal/observability/collector	1.031s
?   	github.com/vdaas/vald/internal/observability/exporter	[no test files]
ok  	github.com/vdaas/vald/internal/observability/exporter/jaeger	0.934s
ok  	github.com/vdaas/vald/internal/observability/exporter/prometheus	0.879s
ok  	github.com/vdaas/vald/internal/observability/exporter/stackdriver	4.926s
ok  	github.com/vdaas/vald/internal/observability/metrics	0.500s
ok  	github.com/vdaas/vald/internal/observability/metrics/agent/core/ngt	1.827s
?   	github.com/vdaas/vald/internal/observability/metrics/grpc	[no test files]
ok  	github.com/vdaas/vald/internal/observability/metrics/manager/compressor	0.568s
ok  	github.com/vdaas/vald/internal/observability/metrics/mem	0.432s
ok  	github.com/vdaas/vald/internal/observability/metrics/runtime/cgo	0.704s
ok  	github.com/vdaas/vald/internal/observability/metrics/runtime/goroutine	0.450s
ok  	github.com/vdaas/vald/internal/observability/metrics/version	0.781s
?   	github.com/vdaas/vald/internal/observability/profiler	[no test files]
ok  	github.com/vdaas/vald/internal/observability/profiler/stackdriver	0.699s
ok  	github.com/vdaas/vald/internal/observability/trace	0.442s
ok  	github.com/vdaas/vald/internal/params	0.551s
ok  	github.com/vdaas/vald/internal/rand	0.297s
ok  	github.com/vdaas/vald/internal/runner	6.895s
ok  	github.com/vdaas/vald/internal/safety	0.485s
ok  	github.com/vdaas/vald/internal/servers	0.854s
ok  	github.com/vdaas/vald/internal/servers/server	0.577s
ok  	github.com/vdaas/vald/internal/servers/starter	0.905s
ok  	github.com/vdaas/vald/internal/singleflight	3.572s
ok  	github.com/vdaas/vald/internal/timeutil	1.109s
ok  	github.com/vdaas/vald/internal/timeutil/location	0.382s
ok  	github.com/vdaas/vald/internal/tls	0.438s
ok  	github.com/vdaas/vald/internal/unit	0.437s
ok  	github.com/vdaas/vald/internal/version	0.413s
ok  	github.com/vdaas/vald/internal/worker	0.394s
```

**Comment by kevindiu** [comment]
LGTM

File: go.sum [hunk 1]
```diff
@@ -832,6 +832,7 @@ golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2/go.mod h1:tRJNPiyCQ0inRvYxb
 golang.org/x/time v0.0.0-20181108054448-85acf8d2951c/go.mod h1:tRJNPiyCQ0inRvYxbN9jk5I+vvW/OXSQhTDSoE431IQ=
 golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 h1:SvFZT6jyqRaOeXpc5h/JSfZenJ2O330aBsf7JfSUXmQ=
 golang.org/x/time v0.0.0-20190308202827-9d24e82272b4/go.mod h1:tRJNPiyCQ0inRvYxbN9jk5I+vvW/OXSQhTDSoE431IQ=
+golang.org/x/time v0.0.0-20191024005414-555d28b269f0 h1:/5xXl8Y5W96D+TtHSlonuFqGHIWVuyCkGJLwGh9JJFs=
```
Comment (hlts2) [inline go.sum:835]: Could you please revert this changes ?

Comment (vankichi) [reply go.sum:835]: Could you tell me why, please?
